pegasus/sites.v3/code.org/views/course_table.haml

```diff
@@ -286,3 +264,102 @@
       %br/
       While listed for younger grades, Courses 2–4 can be used by older students
 
+
+
+- tools = []
+- tools << { name: "Artist", starts: 0, ends: 5, description: "Draw cool pictures and designs with the Artist!", xlong: "Use block programming to create images using commands like moveForward and turnLeft.  The resulting programs and images can be shared.", img: "/images/fill-100x100/tutorials/resubmission/artist.jpg" }
+- tools << { name: "Play Lab", starts: 2, ends: 6, description: "Create a story or make a game with Play Lab!", img: "/images/fill-100x100/playlab_original3.png" }
+- tools << { name: "Game Lab", starts: 7, ends: 9, description: "Create a real-time game with Game Lab.", img: "/shared/images/fill-100x100/courses/logo_gamelab.png"}
+- tools << { name: "Maker Toolkit", starts: 7, ends: 9, description: "Use external hardware for both input and output.", img: "/shared/images/fill-100x100/courses/logo_makerkit.png"}
+- tools << { name: "App Lab", starts: 9, ends: 12, description: "Create a multi-page web app.", img: "/shared/images/fill-100x100/courses/logo_applab.png" }
+- tools << { name: "Text compression widget", starts: 9, ends: 12, description: "Learn about text compression with this interactive widget.", img: "/shared/images/fill-100x100/courses/logo_compression.png" }
+- tools << { name: "Pixelation widget", starts: 9, ends: 12, description: "Learn about graphics foramts with this interactive widget.",img: "/shared/images/fill-100x100/courses/logo_pixelation.png" }
+- tools <<  { name: "Cryptography widget", starts: 9, ends: 12, description: "Learn about cryptography with this interactive widget.", img: "/shared/images/fill-100x100/courses/logo_cryptography.png" }
+- tools << { name: "Internet simulator", starts: 9, ends: 12, description: "A group of students can learn how the internet works by communciating with each other.", img: "/shared/images/fill-100x100/courses/logo_netsim.png" }
+- tools << { name: "Unplugged Activities", starts: 0, ends: 6, description: "A large collection of activities that don't require computers.", link: "/curriculum/unplugged", img: "/shared/images/fill-100x100/courses/logo_unplugged.png" }
+- tools << { name: "Hour of Code", starts: 0, ends: 12, link: "/learn", description: "An interactive catalog of hundreds of hour-long activities.", img: "/images/fill-240x100/learn/student2.png", layout: "wide" }
```

Comment:
We're not using `starts`/`ends` for the table anymore, but maybe we can use it as a subtitle in the expanded box? Grades K-2 or Grades 9-12.